src/registrar/config/urls.py

```diff
@@ -18,6 +24,8 @@
     path("health/", health.health),
     path("edit_profile/", profile.edit_profile, name="edit-profile"),
     path("openid/", include("djangooidc.urls")),
+    path("register/", application_wizard, name="application"),
+    re_path(r"^register/(?P<step>.+)/$", application_wizard, name=APPLICATION_URL_NAME),
```

Comment:
I think this would work fine as

```
    path("register/<step>/", application_wizard, name=APPLICATION_URL_NAME),
```

Reply:
Great catch. I got this from their documentation, but it appears that the `re_path` version is old-fashioned Django style for the much better `path` version.